include/sempr/sempr.hpp

```diff
@@ -14,6 +14,7 @@
 #include <sempr/processing/ActiveObjectStore.hpp>
 #include <sempr/processing/DBUpdateModule.hpp>
 #include <sempr/processing/DebugModule.hpp>
+#include <sempr/processing/PointCloudModule.hpp>
```

Comment:
Optional processing unit, should only be included if needed and not general in the sempr.hpp file.